Testscripts/Windows/GPU-DRIVER-INSTALL.ps1

```diff
@@ -200,6 +200,19 @@ function Main {
             }
         }
 
+        # Install required packages
+        # ---> Ubuntu case
+        if (@("UBUNTU").contains($global:detectedDistro)) {
+        Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort -username $superuser `
+            -password $password -command "install_package libglvnd-dev ubuntu-desktop" -runMaxAllowedTime 1800 -ignoreLinuxExitCode | Out-Null
```

Comment:
source utils.sh?
we can add those into bash script?

Reply:
I don’t think that is good idea because 1) this could be changed per NVidia, 2) one of them installation time takes long.

Reply:
You will hit an issue if invoke install_package directly

Reply:
Done in 3c8f6a60b5a072c1062ab503c0c6a8111da03bd0